addons/point_of_sale/models/pos_order.py

```diff
@@ -472,22 +468,14 @@ def action_view_invoice(self):
             'res_id': self.account_move.id,
         }
 
-    def action_view_refund_orders(self):
+    def action_view_refunded_order(self):
         return {
-            'name': _('Refund Orders'),
-            'view_mode': 'tree,form',
-            'res_model': 'pos.order',
-            'type': 'ir.actions.act_window',
-            'domain': [('id', 'in', self.mapped('lines.refund_orderline_ids.order_id').ids)],
-        }
```

Comment:
I think we should keep this action, don't we ?

Reply:
Yes, this is wrong. It's  fixed in the new commit